packages/compass-import-export/src/components/import-modal.tsx

```diff
@@ -64,7 +65,7 @@ const closeButtonStyles = css({
 type ImportModalProps = {
   isOpen: boolean;
   ns: string;
-  startImport: () => void;
+  startImport: (useNewImportBackend?: boolean) => void;
```

Comment:
This is kinda weird, but temporary. Felt like the cleanest, easiest way to get the feature flag would be a react hook in the component. The component will likely need to branch based off the hook soon enough anyway. And by the end it will just be removed.

Reply:
Actually I'm changing my mind. I'm replacing more and more stuff in another branch and it is becoming a mess. I think I'm going to try to just make a new import store / module next to the old one and then in the components we'll use either the old and the new. Otherwise it is branching sprinkled all over the place.